addons/medical/functions/fnc_setUnconscious.sqf

```diff
@@ -94,8 +94,16 @@ _unit setUnitPos "DOWN";
 if (GVAR(moveUnitsFromGroupOnUnconscious)) then {
     [_unit, true, "ACE_isUnconscious", side group _unit] call EFUNC(common,switchToGroupSide);
 };
+// Delay Unconscious so the AI dont instant stop shooting on the unit #3121
+if (GVAR(delayUnconCaptive) == 0) then {
+    [_unit, QGVAR(unconscious), true] call EFUNC(common,setCaptivityStatus);
+} else {
+    [{
+        [_this select 0, QGVAR(unconscious), true] call EFUNC(common,setCaptivityStatus);
```

Comment:
Need to make sure they are still unconc at this point, if delayUnconCaptive > (wake up time)